rollup/sync_service_test.go

```diff
@@ -237,62 +236,16 @@ func TestSyncServiceQueueBatchAppend(t *testing.T) {
 	<-service.doneProcessing
 	rtx, _ := service.txCache.Load(queueIndex.Uint64())
 
+	if rtx == nil {
+		t.Fatal("Unable to process tx")
+	}
 	ok, err := txProcessed(t, rtx, service)
 	if !ok {
 		t.Fatal(err)
 	}
 }
 
 func txProcessed(t *testing.T, rtx *RollupTransaction, service *SyncService) (bool, error) {
```

Comment:
This is all removed due to the mempool no longer being involved in the codepath. Instead we should be checking that the tx ended up in the chain